test/e2e/v2/backuprestore/README.md

```
@@ -148,6 +148,166 @@ spec:
+      defaultPlugins:
+        - openshift
+        - aws
+        - kubevirt
+        - csi
+      noDefaultBackupLocation: true
+      logLevel: debug
+EOF
+```
+
+Note: For OpenShift ADP 1.5+, the customPlugins might be left out. Instead, the hypershift plugin can be listed directly under defaultPlugins as follows:
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)